src/Models/User.php

```diff
@@ -173,7 +185,10 @@ public function getSiteMemberships()
     public function getSSHKeys()
     {
         if (empty($this->ssh_keys)) {
-            $this->ssh_keys = $this->getContainer()->get(SSHKeys::class, [['user' => $this,],]);
+            $this->getContainer()
+                ->add('SshKeys', SSHKeys::class)
+                ->addArgument(['user' => $this]);
+            $this->ssh_keys = $this->getContainer()->get('SshKeys');
```

Comment:
@TeslaDethray @greg-1-anderson 
The good news is, after updating everything commands are starting to work again. The bad news is, every container call has to be re-written and the new syntax is much more verbose.